### ISSUE TITLE
MacOS build

### ISSUE THREAD

**Issue by f8ttyc8t** [wereturtle/ghostwriter#175]
Apart from [official documentation](https://github.com/wereturtle/ghostwriter), steps below helped me to produce running app:

# Quick guide

## 1. Install QT via homebrew:
It seems QT 5.5 is required, therefore you need (based on hint [number5](https://github.com/thoughtbot/capybara-webkit/issues/885#issuecomment-199502619)) to add this version to brew:

~~`$ brew tap homebrew/versions`~~ [(deprecated)](https://github.com/homebrew/homebrew-core)
`$ brew install qt55`
`$ brew link --force qt55`

## 2. Make qmake available by command line:
`$ PATH=$PATH:/usr/local/opt/qt@5.5/bin`
`$ export PATH`

## 3. (Fix for official guide)
`cp resources/mac/Info.plist resources/`
(`cp` command is missing in documentation)

## 4. Build application:
Now, everything should be set up. Issue already described commands:
`$ qmake -spec macx-g++`
`$ make`

## 5. The rest remains like already [described](https://github.com/wereturtle/ghostwriter).

# Problems??
## 1.) If you get an error like below:
> Error: Could not resolve SDK path for 'macosx'

probably tip below (based on hint [rsp1984](https://stackoverflow.com/a/41524934))
`$ sudo xcode-select -s /Applications/Xcode.app/Contents/Developer`
might help.

## 2.) If you get an error like:
> Xcode not set up properly. You may need to confirm the license agreement by running /usr/bin/xcodebuild.

this tip from [Amin](http://amin-ahmadi.com/2016/09/26/qt-creator-issue-on-mac-os-x-fix-xcodebuild-issue-with-xcode-8-0/) helped me...

Good luck building this great, great gem!

**Comment by Phara0h** [comment]
Followed steps 1 through 3 and got it compiled and running perfectly. The only problem I ran into instantly is I can only export as HTML. Guessing the code is not checking for bsd/mac dumb locations for install of bins.

**Comment by f8ttyc8t** [comment]
I see - guess you are right. But I have no idea how to handle that yet.

**Comment by f8ttyc8t** [comment]
Checked it. I think you need to to install e.g. **pandoc** also.

At best, utilize **homebrew** again:
>$ brew install pandoc

Now you should be able to select "Pandoc" as "Markdown Converter" in Export dialog.

**Comment by f8ttyc8t** [comment]
@Phara0h, did instructions (especially pandoc hint) work for you?

**Comment by AuroraMartell** [comment]
I tried downloading the .exe file and running it through WineBottler, but it's all _buggy_. For those interested on a MacOS build of this app, I'm happy to announce I am in the process of developing a Mac alternative for this app.

**Comment by dominikfranz** [comment]
@f8ttyc8t
For me compiling the app worked fine with your instructions, unfortunately pandoc didn't show up in ghostwriter after installing pandoc with homebrew though...

**Comment by AuroraMartell** [comment]
Couldn't someone take the code and make a Mac version?

**Comment by DiggidyDale** [comment]
was there ever a fix of pandoc on Mac?

**Comment by iamandrewluca** [comment]
Please update `qt` formula install. This is last one with `--with-qtwebkit` option
```
brew install qt@5.7 --with-qtwebkit
```

**Comment by drscotthawley** [comment]
@iamandrewluca Even that version doesn't seem to work anymore: 
```
$ brew install qt@5.7 --with-qtwebkit
Error: No available formula with the name "qt@5.7" 
==> Searching for a previously deleted formula (in the last month)...
Warning: homebrew/core is shallow clone. To get complete history run:
  git -C "$(brew --repo homebrew/core)" fetch --unshallow

Error: No previously deleted formula found.
==> Searching for similarly named formulae...
==> Searching local taps...
Error: No similarly named formulae found.
==> Searching taps...
==> Searching taps on GitHub...
Error: No formulae found in taps.

$ git -C "$(brew --repo homebrew/core)" fetch --unshallow
remote: Counting objects: 258120, done.
remote: Compressing objects: 100% (119850/119850), done.
remote: Total 258120 (delta 137937), reused 255611 (delta 135481), pack-reused 0
Receiving objects: 100% (258120/258120), 57.41 MiB | 480.00 KiB/s, done.
Resolving deltas: 100% (137937/137937), completed with 3320 local objects.


$ brew install qt@5.7 --with-qtwebkit
Error: No available formula with the name "qt@5.7" 
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching for similarly named formulae...
==> Searching local taps...
Error: No similarly named formulae found.
==> Searching taps...
==> Searching taps on GitHub...
Error: No formulae found in taps.

```
